src/pokemon.c

```diff
@@ -6864,3 +6887,37 @@ const u8 *GetMoveAnimationScript(u16 moveId)
     }
     return gMovesInfo[moveId].battleAnimScript;
 }
+
+void UpdateDaysPassedSinceFormChange(u16 days)
+{
+    u32 i;
+    for (i = 0; i < PARTY_SIZE; i++)
+    {
+        struct Pokemon *mon = &gPlayerParty[i];
+
+        if (GetMonData(mon, MON_DATA_SPECIES, 0))
```

Comment:
you could invert this to save some depth. would make the code a bit more readable.